docroot/modules/custom/layout_builder_custom/layout_builder_custom.module

```diff
@@ -475,6 +475,13 @@ function layout_builder_custom_form_node_page_layout_builder_form_alter(&$form,
   $form['#attached']['library'][] = 'layout_builder_custom/layout_builder_custom.overrides';
 }
 
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function layout_builder_custom_form_node_private_page_layout_builder_form_alter(&$form, FormStateInterface $form_state, $form_id) {
```

Comment:
can this move to sppa_core as `sppa_core_form_node_private_page_layout_builder_form_alter()` since this is a one-off for now?